src/components/App/App.js

```diff
@@ -31,14 +35,27 @@ function App() {
   const [isSuccessRegistration, setSuccessRegistration] = useState(false);
 
   //Status
+  const [jwt, setJwt] = useState(localStorage.getItem("jwt"));
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [keyword, setKeyword] = useState("");
   const [searchKeywords, setSearchKeywords] = useState([]);
+  const [globalErrorMessage, setGlobalErrorMessage] = useState("");
 
   const [currentUser, setCurrentUser] = useState({});
   const [cards, setCards] = useState([]);
   const [savedCards, setSavedCards] = useState([]);
 
+  const BASE_URL = "https://api.news-leo.mooo.com";
+  // const BASE_URL = "http://localhost:3000";
+  const mainApi = new MainApi({
+    baseUrl: BASE_URL,
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+      "Access-Control-Allow-Origin": BASE_URL,
+      authorization: `Bearer ${jwt}`,
+    },
+  });
```

Comment:
Please, move it inside `MainApi.js` to create the innstance only **once** and export it from there


Now you create an instance every time `App` rerenders